terraform/envs/production/terraform.tfvars

```diff
@@ -33,4 +33,7 @@ google_cloudbuild_trigger_filename = "production.cloudbuild.yaml"
 google_cloudbuild_trigger_name = "version-manual-deploy"
 github_owner = "amplication"
 github_name = "amplication"
-github_tag = "v.+"
\ No newline at end of file
+github_tag = "v.+"
+
+# Apps Cloud Build
+apps_gcp_project_id = "amplication-production"
```

Comment:
I think the data on the deployment environment should be taken from a list in the DB. initially it will be a table with a single record (different between prod, staging, and dev) - but in the future we will be able to have multiple projects and select the right one dynamically.

Reply:
I do not agree. This kind of configuration should come from the environment. If we would like to have multiple ones we can provide either by providing a list in the environment or multiple variables